var/spack/repos/builtin/packages/kubernetes/package.py

```diff
@@ -12,14 +12,46 @@ class Kubernetes(Package):
     for deployment, maintenance, and scaling of applications."""
 
     homepage = "https://kubernetes.io"
-    url = "https://github.com/kubernetes/kubernetes/archive/v1.19.0-alpha.0.tar.gz"
+    url = "https://github.com/kubernetes/kubernetes/archive/refs/tags/v1.26.3.tar.gz"
```

Comment:
Is there a reason to use an autogenerated `url` instead of:
```suggestion
    url = "https://github.com/kubernetes/kubernetes/releases/download/v1.27.0/kubernetes.tar.gz"
```
?

Reply:
Ah! I tried this at first too! Unfortunately that `kubernetes.tar.gz` that they publish as a release asset doesn't include Makefile metadata needed to build the project. (Ugh Kubernetes! Why do you do this!)